master/security/service-accounts.md

```diff
@@ -87,6 +87,29 @@ spec:
   selector: 'app == "db"'
 ```
 
+#### Restrict label assignment with service account selectors
+
+Network policies can be applied to endpoints using selectors that match labels on either the endpoint itself, the endpoint's namespace, or the endpoint's service account. By specifying selectors based on the endpoint's service account we can employ Kubernetes RBAC to limit which users are allowed to apply labels. In the following example, pods with an **intern** service account will only be allowed to communicate with other pods with service accounts labeled `role: intern`.
```

Comment:
Wondering if we should spell this out a bit more clearly, since it's a sorta complicated concept.

e.g., Turn this statement: "Kubernetes RBAC to limit which users are allowed to apply labels"

Into this one:

"Kubernetes RBAC to control which users can assign labels to service accounts. That group may be separate from the group of users who are allowed to deploy pods."

Reply:
(And probably similar updates in the namespace doc)

Reply:
Fixed